service/sonar-project.properties

```diff
@@ -0,0 +1,3 @@
+# Inheritance tree of classes should not be too deep
```

Comment:
Reminder to find out and tell people what sonar is

Reply:
Update: It's a code quality checker

Reply:
This comment should probably have linked to the exact sonar rule, which is https://rules.sonarsource.com/java/RSPEC-110

Reply:
Added this link as a comment in the file